### DISCUSSION TITLE
Add support attributes for each product variants to customize it

### DISCUSSION THREAD

**Discussion by devmansurov** [lunarphp/lunar#64] [Feature Requests]
In some cases, besides the basic product information, there are additional data for each product variant. To do this, you need to provide access to adding attributes to product variants. I think adding such an opportunity is not difficult for you, since attributes can be used in any table based on your database structure.

**Comment by glennjacobs** [comment]
It’s coming in the next release ;-) Just going through testing at the mo.

**Reply by devmansurov** [reply]
I understood you. Thank you. Let's wait for new releases.
